tools/util.wdl

```diff
@@ -95,6 +103,8 @@ task split_string {
         Int disk_size = 1
     }
     command <<<
+        set -euo pipefail
+
         echo ~{input_string} | sed 's/~{delimiter}/\n/g' > output.txt
```

Comment:
I suppose a delimiter of `/` would break this. Admittedly, I think I wrote this in the first place. I wonder if it should change to something like `| tr '~{delimiter} '\n'`.